src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml

```diff
@@ -51,65 +52,64 @@
 			</ListView.ItemContainerStyle>
 			<ListView.ItemTemplate>
 				<DataTemplate x:DataType="fs:RecentItem">
-					<ListViewItem RightTapped="ListViewItem_RightTapped" ToolTipService.ToolTip="{x:Bind RecentPath}">
```

Comment:
It's possible that having a ListViewItem inside the ListViewItem template broke virtualization. In retrospect I should have caught this when the initial change was implemented.

Reply:
Yes. I did not check it enough. I have fixed it with reference to the implementation in the column layout.